src/theme.ts

```diff
@@ -3,6 +3,26 @@ import { createTheme } from "@mui/material";
 const { breakpoints } = createTheme();
 
 export const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#3023b8",
+      light: "#9b93f1",
+      dark: "#080155",
```

Comment:
Is there currently a way to switch the theme on the site?

Reply:
No, there is no light/dark theme option currently on the site. Those are just three selections from the design system primary colors. Really, I just needed primary.main for the ProgressBar component, but it might be handy to have those in the future. I modeled it after this: [mui](https://v3.mui.com/customization/themes/)